### ISSUE TITLE
Unable to resize font

### ISSUE THREAD

**Issue by HaydenReeve** [earlSt1/vtt-custom-nameplates#15]
Hello there 👋

I don't appear to be able to resize the font on the bar, or actually be tinker with several options at all.

I appear to have correctly installed all dependencies, and do not have many modules running. I have disabled everything except this as libwrapper.

I've checked my console, and I get the following error:

```
caught (in promise) Error: undefined. Unable to convert color 
[Detected 1 package: custom-nameplates]
    at $e.normalize (index.mjs:3:37)
    at set value [as value] (index.mjs:3:37)
    at $e.setValue (index.mjs:3:37)
    at On (earcut.js:242:1)
    at set fill [as fill] (earcut.js:242:1)
    at _mergeInsert (commons.js:2787:17)
    at CustomNameplates.mergeObject (commons.js:2761:12)
    at CustomNameplates.setCanvasStyleTo (custom-nameplates.js:99:18)
    at CustomNameplates.setCanvasStyle (custom-nameplates.js:92:22)
    at registerSettings (custom-nameplates.js:261:27)
    at Object.fn (custom-nameplates.js:295:11)
    at #call (foundry.js:730:20)
    at Hooks.callAll (foundry.js:687:17)
    at Game.setupGame (foundry.js:8712:11)
    at async Game._initializeGameView (foundry.js:9990:5)
```

**Comment by earlSt1** [comment]
Hi, sorry for the long response. This looks like there was a bug with the default value for the global style. I'll push an update to the v11 branch soon but running this macro should fix it:

```js
await game.settings.set("custom-nameplates", "global-style", {
    fontSize: 24,
    fontFamily: "Signika",
    fontColor: "#FFFFFF",
    shadowColor: "#000000",
    strokeColor: "#111111",
});
```

**Comment by earlSt1** [comment]
This should be fixed in v2.1.5 / v2.2.0. Closing for now
